packages/pwa/src/lib/utils.ts

```
@@ -0,0 +1,6 @@
+import { clsx, type ClassValue } from "clsx"
+import { twMerge } from "tailwind-merge"
+
+export function cn(...inputs: ClassValue[]) {
```

ESLint

[error] Missing return type on function.

(@typescript-eslint/explicit-function-return-type)